virtool/groups/utils.py

```diff
@@ -19,9 +24,42 @@ def merge_group_permissions(groups: List[dict]) -> Dict[str, bool]:
     for p in Permission:
         for group in groups:
             try:
-                if group["permissions"][p.value]:
+                if group["permissions"][p.value.id]:
```

Comment:
```suggestion
                if group["permissions"][p.value]:
```
Use p.value for now since using the legacy Permission enum.